app/templates/src/main/resources/config/liquibase/changelog/_initial_schema.xml

```diff
@@ -173,6 +173,36 @@
                                  constraintName="fk_evt_pers_audit_evt_data"
                                  referencedColumnNames="event_id"
                                  referencedTableName="jhi_persistent_audit_event"/>
+    <% if (enableSocialSignIn) { %>
+    <!-- Social -->
+    <createTable tableName="jhi_UserConnection">
+          <column name="userId" type="varchar(255)">
```

Comment:
this should be lower case, "user_id" -> can you configure those? I don't know if those come from Spring Social